defaults/main.yml

```diff
@@ -1,30 +1,12 @@
 ---
-rkt_version: 1.30.0
-rkt_fly_version: 1.30.0
-node_exporter_version: v0.17.0
-
-versions:
-  restund: 0.4.14w7b1.0.47
-  nginx:   1.15.12-alpine
-  rex:     0.2.2b1.0.6
-  process_exporter: 0.4.0
-
-restund_user: restund
-restund_udp_listen_port: 3478
-restund_tcp_listen_port: 3478
-restund_tls_listen_port: 5349
+restund_version: 0.4.14w7b1.0.47
+restund_user: restund # This also create the same group
+restund_udp_listen_port: 80
```

Comment:
We changed these defaults; because I if I understand correctly listening on these ports is beneficial in almost all cases for TURN

Reply:
Port 80 is required for certbot, and nginx listens on that port (during certificate requests and renewals) so I think this doesn't work to use port 80 here.

Defaulting tls to 443 should work fine.

Reply:
This comment is supposed to be for `restund_tcp_listen_port`, isn't it?

We don't run certbot inside k8s in that same network namespace, so for the current setup I'd propose just setting other defaults.

Reply:
I'vde decided to not change the defaults. These port numbers are the ones as documented in the RFC for STUN/TURN